README.md

```diff
@@ -67,7 +67,7 @@ Optionally add an NPM run script to package.json
 ## HTML Example Demo
 
 ```html
-<h1 class="h2">Hamburger</h1>
+<h1 class="h1">Hamburger</h1>
```

Comment:
Seems to be working for me. `.h2` is just a utility to change the font size while leaving the `<h1>` semantically correct. See: https://github.com/jxnblk/doctor-mark/blob/master/index.html#L71

Maybe file a separate issue to see if I can reproduce?